src/index.js

```diff
@@ -33,8 +33,9 @@ const ApolloClientPlugin = createPlugin({
     fetch: FetchToken,
     authKey: ApolloClientAuthKeyToken.optional,
     schema: GraphQLSchemaToken.optional,
+    context: ApolloContextToken.optional,
```

Comment:
Given that we often rely on "Koa Context", i would like to rename the service mapping here. What do you think about something called `apolloContext`?

Reply:
Agreed - I've updated to `apolloContext`.

Reply:
Thanks, I think we can change the `context` key here as well to `apolloContext`, it's essentially just a local name mapping for the dependency. We would need to make the change on line 38 as well.